src/views/esp/view_esp.cpp

```diff
@@ -81,42 +81,45 @@ namespace big
 			}
 
 			draw_list->AddText(name_pos, esp_color, name_str.c_str());
+			const bool in_god = ped_damage_bits & (uint32_t)eEntityProofs::GOD;
 			std::string mode_str = "";
 			if (g.esp.god)
 			{
-				if (ped_damage_bits & (uint32_t)eEntityProofs::GOD)
+				if (inGod)
```

Comment:
Imagine compiling before committing bruh, what a waste of my time....

Reply:
Wha?

Reply:
1. I requested the change to snake case
2. You change it only in 1 spot
3. Commit without compiling locally
4. Build fails because you didn't change it to snake case everywhere

Reply:
@Yimura You can always tell the extra shitty commits when:

>This commit was created on GitHub.com and signed with GitHub’s verified signature.